packages/pwa/app/components/_app-config/index.jsx

```diff
@@ -88,8 +77,16 @@ const AppConfig = ({children, locals = {}}) => {
 
 AppConfig.restore = (locals = {}) => {
     // Parse the locale from the page url.
-    const locale = getLocale(locals) || DEFAULT_LOCALE
-    const currency = getPreferredCurrency(locale) || DEFAULT_CURRENCY
+    const originalUrl = locals.originalUrl
+    const url = pathToUrl(originalUrl)
+    const site = resolveSiteFromUrl(url)
+
+    apiConfig.parameters.siteId = site?.id
```

Comment:
Add a comment `// Update the commerce api 'siteId' value`. 
Also, is using the optional notion required here?

Would something like this be easier to read and understand what you are doing?

```
// Update client config with parsed siteId.
if (site) {
    apiConfig.parameters.siteId = site?.id
}
```